svc_translation.go

```diff
@@ -111,7 +114,7 @@ func (t *NewTranslation) UnmarshalJSON(data []byte) error {
 
 type UpdateTranslation struct {
 	Translation                string   `json:"translation"`
-	IsFuzzy                    *bool    `json:"is_fuzzy,omitempty"`
```

Comment:
Breaking changes

Reply:
Yes, because the current version is invalid and won't work with Lokalise Api